wagtailflags/templates/wagtailflags/includes/header.html

```diff
@@ -1,18 +1,9 @@
-{% load wagtailadmin_tags wagtailflags_admin %}
+{% extends 'wagtailadmin/shared/header.html' %}
+{% load i18n modeladmin_tags wagtailadmin_tags wagtailflags_admin %}
 
-{% if flag %}
-  <ul class="breadcrumb flags-breadcrumb">
```

Comment:
With the removal of the only use of `flags-breadcrumb` here you should be able to delete its definition:

https://github.com/cfpb/wagtail-flags/blob/a25621c6be18e5f68b4715655c5301c32f67e4b7/wagtailflags/static/wagtailflags/css/wagtailflags.css#L1-L3